### DISCUSSION TITLE
How to enable missing option Print Save as PDF in CefSharp

### DISCUSSION THREAD

**Discussion by mukeshmishra996** [cefsharp/CefSharp#4758] [Q&A]
Hi Team,

While attempting to save a page as a PDF using the "Print as PDF" feature, I've noticed that not all page hyperlinks are clickable. I'm interested in a "Save as PDF" feature similar to that in Chrome/Edge. Can anyone suggest how I might achieve this functionality in CefSharp?

<img width="942" alt="image" src="https://github.com/cefsharp/CefSharp/assets/14114368/049509a4-aea9-44f8-b4a1-42c341ab50c1">

**Comment by amaitland** [comment]
> Can anyone suggest how I might achieve this functionality in CefSharp?

CEF would need to add support for this. Issue tracker at https://github.com/chromiumembedded/cef

If you are new to the project then please read the first few paragraphs of https://github.com/cefsharp/CefSharp/wiki/General-Usage

You can try switching to the Chrome Runtime see https://github.com/cefsharp/CefSharp/discussions/4123

**Reply by aimcosoftware** [reply]
You could embed [PDF.js,](https://mozilla.github.io/pdf.js/) which then gives you full control over PDF files.
